impl/src/main/java/org/ehcache/internal/events/CacheEventNotificationServiceProviderConfiguration.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.internal.events;
+
+import org.ehcache.events.CacheEventNotificationListenerServiceProvider;
+import org.ehcache.spi.service.ServiceConfiguration;
+
+/**
+ * @author palmanojkumar
+ *
+ */
+public class CacheEventNotificationServiceProviderConfiguration implements ServiceConfiguration<CacheEventNotificationListenerServiceProvider> {
```

Comment:
Currently this class is not used anywhere. This config class is intended for service  lookup using ServiceProvider#findServiceFor method.